### DISCUSSION TITLE
Add missing collectibles to the API

### DISCUSSION THREAD

**Discussion by HammyInTheWall** [HypixelDev/PublicAPI#484] [Ideas]
I've been working on a collectibles tracker that tracks your percentages of all collectibles owned, as well as specific lists of which ones you own (plus documenting all the info I can find on collectibles). Information on collectibles in the API is already kinda janky (misspelled names, mislabeled sections, etc), but there are a few collectibles that are outright not even in the API. They seem to be certain cosmetics bought from the in-game store or store.hypixel.net. There is no information about the anywhere in the public API so it's impossible to create an accurate list of owned collectibles, nor an accurate percentage of total collectibles owned.

Also, there are seemingly items that don't exist that are in the API? For example, `player.vanityMeta.packages."particlepack_red_dust";` is visible when looking at my owned collectibles in my API page, yet that isn't actually correlated to any existing particle pack on the server. I've been keeping track of any issues with collectibles in the API so if anyone actually does plan on fixing it (which I doubt because nobody but me seems to care much about collectibles and pets in the first place) feel free to get in contact with me because I have a list of stuff that needs to be added, removed, or changed.

**Comment by eatmyvenom** [comment]
welp u tried
